index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/font-awesome.min.css"></head>
+    <script src="https://kit.fontawesome.com/7415ea4535.js" crossorigin="anonymous"></script>
+<body>
+
+    <header>
+        <h3>Welcome</h3>
+        <i class="fa-solid fa-bars small"></i>
+    </header>
+
+    <section class="intro">
+        <h2 class="header">I'm <span>Stephen Katuli</span>, nice to meet you</h2>
+        <p>Greetings, I am a professional software developer, proficiency in Javascript, HTML and CSS and Laravel.</p>
+        <p>I am passionate about starting my professional development and making meaningful contributions to cutting-edge projects.</p>
+    </section>
+
+    <div class="contacts">
+        <a href="https://github.com/stephenkati"><i class="fa-brands fa-github"></i></a>
+        <a href="https://www.linkedin.com/in/stephen-katuli-a92752251/"><i class="fa-brands fa-linkedin"></i></a>
+        <a href="https://twitter.com/Stephen57913145"><i class="fa-brands fa-twitter"></i></a>
+    </div>
```

Comment:
- [ ] I think your social media links should be a list of icons a use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).  Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href Like this
  ```html
   <ul>
        <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
        <!-- other links here -->
   </ul>
  ```